src/olympia/lib/settings_base.py

```diff
@@ -1511,7 +1511,7 @@ def read_only_mode(env):
     # Extract tag of the root path of the API route
     # /api/v5/foo -> foo
     'SCHEMA_PATH_PREFIX': '/api/v[0-9]',
-    'TITLE': 'AMO API',
+    'TITLE': 'AMO API (Experimental)',
     'DESCRIPTION': 'Addons Server API Documentation',
```

Comment:
Maybe even something like:

```suggestion
    'DESCRIPTION': 'Addons Server API Documentation (Experimental, check out <a href="https://mozilla.github.io/addons-server/topics/api/index.html">official documentation</a> for a more reliable source of information)',
```

+ formatting to appease `ruff` ? Or keep `(Experimental)` in the title but find a way to stick the link to the official docs somewhere ?